cocos2d/tilemap/CCSGTMXLayer.js

```diff
@@ -56,11 +56,15 @@
  * @property {Number}               tileHeight          - Height of a tile
  */
 _ccsg.TMXLayer = cc.SpriteBatchNode.extend(/** @lends _ccsg.TMXLayer# */{
```

Comment:
A complete rewrite of TMXLayer, all vertex data is generated dynamically, with culling, it's faster than before